### PR TITLE
Use common and specific conf. Allow to custom conf at deployment (prefix, names, params...).

### PR DESCRIPTION
**Pull request by sBouzols**
Signed-off-by: sBouzols <sylvain.bouzols@gmail.com>

### CODE REVIEW

File: k8s/base/config/common-application.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  # RabbitMQ binding destination prefixes
+  rabbitmq:
+    destination:
+      prefix: ${powsybl-ws.environment:}
+
+  # elasticsearch index prefixes
+  elasticsearch:
+    index:
+      prefix: ${powsybl-ws.environment:}
```
Comment (geofjamg) [inline k8s/base/config/common-application.yml:22]: Add a new line

File: k8s/overlays/azure-dev/common-application.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+powsybl-ws:
+  environment: dev_
```
Comment (geofjamg) [inline k8s/overlays/azure-dev/common-application.yml:2]: Don't break compatibility, right now as we do not intend to deploy a second environment on Azure

File: k8s/overlays/azure-integ/common-application.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+powsybl-ws:
+  environment: integ_
```
Comment (geofjamg) [inline k8s/overlays/azure-integ/common-application.yml:2]: Ok but we don't have integ environment is not deployed on Azure

File: k8s/overlays/local/common-application.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+powsybl-ws:
+  environment: local_
```
Comment (geofjamg) [inline k8s/overlays/local/common-application.yml:2]: When we deploy the application locally using minikube, do we want to share the database with the one we use for day to day dev (docker compose one) ? So for you the answer is no?

**Review by geofjamg — CHANGES_REQUESTED** [review]

File: k8s/overlays/local/common-application.yml (hunk 4 above)
Comment (sBouzols) [reply k8s/overlays/local/common-application.yml]: removed

File: k8s/overlays/azure-dev/common-application.yml (hunk 2 above)
Comment (sBouzols) [reply k8s/overlays/azure-dev/common-application.yml]: removed

**Pull request merged**